web-app/src/lib/actions/stripe.actions.ts

```diff
@@ -0,0 +1,81 @@
+"use server";
+
+import { headers } from "next/headers";
+import Stripe from "stripe";
+
+import { getEnvSecrets } from "@/config/env.config"; // Ensure this path is correct
+
+const stripe = new Stripe(getEnvSecrets().STRIPE_SECRET_KEY, {
+  apiVersion: "2025-03-31.basil", // Corrected API version
```

Comment:
should we hardcode this?

Reply:
It's actually not needed, because the latestVersion is this one...
59cd311215ea8dfef4330a5c1f4b88c680a51f0c